framework/core/hpp_command_buffer.h

```diff
@@ -206,7 +206,7 @@ class HPPCommandBuffer : public core::HPPVulkanResource<vk::CommandBuffer>
 	void flush_descriptor_state(vk::PipelineBindPoint pipeline_bind_point);
 
 	/**
-	 * @brief Flush the piplines state
+	 * @brief Flush the pipelines state
```

Comment:
nit: Since these are corrections, this should probably either be "pipeline" or "pipeline's"

Reply:
Thank you! The function name is `flush_pipeline_state` so I changed it to `pipeline state`.